ckan/views/dataset.py

```diff
@@ -1084,13 +1084,13 @@ def history(package_type, id):
     format = request.args.get(u'format', u'')
     if format == u'atom':
         # Generate and return Atom 1.0 document.
-        from webhelpers.feedgenerator import Atom1Feed
-        feed = Atom1Feed(
+        from feedgen.feed import FeedGenerator
+        feed = FeedGenerator(
```

Comment:
`FeedGenerator` accepts no arguments, that is why the tests are failing. This piece of code and the tests an functions depending on the how Atom1Feed get initialized should be changed.